plugins/single_plugins/move.cpp

```diff
@@ -303,8 +303,22 @@ class wayfire_move : public wf::plugin_interface_t
             return;
         }
 
-        /* Snap the view */
-        if (enable_snap && (slot.slot_id != 0))
+        /**
+         * Snap the view
+         * A view that is fullscreened may break if resized i.e
+         * tiled_vertical or tiled_horizontal, games usually have
+         * their own option to be fullscreen.
+         * If the compositor resizes them and thus removes
+         * for xwayland _NET_WM_STATE fullscreen (there
+         * are not much wayland games around to test what wayland games
+         * do when they have fullscreen in their settings and
+         * the compositor puts them in another state) they break.
+         *
+         * This way fullscreened view can be moved from one
+         * output to another and be restored to fullscreen
+         * on a different output (hopefully with the same resolotion)
+         */
+        if (enable_snap && (slot.slot_id != 0) && !this->view->fullscreen)
```

Comment:
why do we need to check this here, below you say that the slot for fullscreen views shouldn't be updated, so it will always be 0 anyway for fullscreen views?

Reply:
removed: new PR upcoming old repo deleted